sdk/translation/Azure.AI.Translation.Document/src/TranslationSource.cs

```diff
@@ -19,18 +19,48 @@ public partial class TranslationSource
         [CodeGenMember("SourceUrl")]
         public Uri SourceUri { get; }
 
-        /// <summary>
-        /// The set of options that can be specified to filter the documents by name
-        /// using prefix and suffix.
-        /// </summary>
-        [CodeGenMember("Filter")]
-        public DocumentFilter Filter { get; set; }
-
         /// <summary>
         /// Language code for the source documents.
         /// If none is specified, the source language will be auto-detected for each document.
         /// </summary>
         [CodeGenMember("Language")]
         public string LanguageCode { get; set; }
+
+        /// <summary>
+        /// A case-sensitive prefix string to filter documents in the source path for translation.
+        /// For example, when using a Azure storage blob Uri, use the prefix to restrict sub folders for translation.
+        /// </summary>
+        public string Prefix
+        {
+            get => Filter.Prefix;
```

Comment:
Both here and in the `Suffix` property, we should do a null check, right?
```cs
get => Filter?.Prefix;
```

Otherwise we'll get a null ref exception when trying to get these properties if no filter is set.